modules/codemod/lib/v6/moveAndRenameSearchBar.ts

```diff
@@ -42,15 +42,25 @@ const transform: Transform = (file, api) => {
     nodePath.insertBefore(j.importDeclaration(specifiers, j.stringLiteral(sourcePath)));
   }
 
+  const discoveredImportAliases: Record<string, boolean> = {};
```

Comment:
I'm going to rename this. I want to add another object map to keep track of aliases we don't rename cases like this:
```
import { Header as CanvasHeader } from '@workday/canvas-kit-labs-react/header';

const SomeComponent = () => {
  return <>
    <Header />
    <CanvasHeader />
  </>;
};
 ```